tests/test-data/bridges/externalComponentBridge.ts

```diff
@@ -0,0 +1,8 @@
+import { IExternalComponent } from '../../../src/definition/externalComponent';
+import { IExternalComponentBridge } from '../../../src/server/bridges';
+export class TestExternalComponentBridge implements IExternalComponentBridge {
+    // TODO Need to add the actual tests here
```

Comment:
Looking forward to seeing these tests. :)

Reply:
We finally use AppExternalComponentManager to maintains all the external components itself. Not depends on the bridged system anymore. This file should be removed.

Reply:
resolved in https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/149/commits/6a522751da78a0ab521b1b1e34587ea0c513d5f7